kube-config-files/list-notifications-rw-service.yaml

```diff
@@ -0,0 +1,72 @@
+###############################################################################
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: list-notifications-rw
+  labels:
+    app: list-notifications-rw
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: list-notifications-rw
+  template:
+    metadata:
+      labels:
+        app: list-notifications-rw
+        visualize: "true"
+    spec:
+      containers:
+        - name: list-notifications-rw
+          image: coco/list-notifications-rw:v0.1.1
+          resources:
+            limits:
+              memory: 128Mi
+          env:
+            - name: MONGO_ADDRESSES
+              value: "mongodb-0.mongodb.default.svc.cluster.local:27017,mongodb-1.mongodb.default.svc.cluster.local:27017,mongodb-2.mongodb.default.svc.cluster.local:27017"
+            - name: DUMP_REQUESTS
+              value: "false"
+            - name: API_HOST
+              value: "test.api.ft.com"
+            - name: CACHE_TTL
+              value: "10"
```

Comment:
We should demonstrate that it is possible to be done with config maps, and after that we should just hardcode it and put a comment that it should be moved to config maps.